features/step_definitions/analytics_steps.rb

```diff
@@ -1,7 +1,7 @@
 Then(/^the links on the page have tracking attributes$/) do
   visit finder_path("government/policies/benefits-reform")
 
-  expect(page).to have_selector('.finder-results[data-module="gem-track-click"]')
+  expect(page).to have_selector('.js-live-search-results-block[data-module="gem-track-click"]')
 
```

Comment:
Does the query selector here need to be so specific? Perhaps it could be changed to `js-live-search-results-block`. i think there is only one occurrence of this selector.

Reply:
I think the intent of this test is to check that this element both exists and includes the `gem-track-click` module, so I think the specific selector is intentional.

Reply:
Ah yes, my mistake - I see it's mentioned in the test description 👍